deals/rpc/rpc.go

```diff
@@ -147,7 +147,7 @@ func (s *RPC) Watch(req *WatchRequest, srv RPCService_WatchServer) error {
 			PieceCid:      update.PieceCID.Bytes(),
 			Size:          update.Size,
 			PricePerEpoch: update.PricePerEpoch,
-			MinDuration:   update.Duration,
+			Duration:      update.Duration,
```

Comment:
Ah right because DealInfo is the actual state of the deal, not the desired config